pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>${p.type}</packaging>
 
     <properties>
-        <models.version>7.1.7</models.version>
+        <models.version>7.0</models.version>
```

Comment:
stupid answer: because this number has to be aligned with the number in `builds.json` which happens to be 7.0. This was failed to be updated whenever we move from 7.0 to 7.1. Not sure when that happened, I have been too lazy to change this as it has implications on how to use the whole library.